theme/views.py

```diff
@@ -1,5 +1,7 @@
 from json import dumps
 
+from django_comments.models import Comment
+from django_comments.views.moderation import perform_delete
```

Comment:
Nice to be able to leverage perform_delete function directly from django_comments module.